app/lib/websocket.js

```diff
@@ -0,0 +1,7 @@
+import websocket from "websocket-stream";
+
+export default function createWebsocket(opts) {
+  opts = opts || {};
+  const url = opts.url || "ws://10.0.2.2:3000";
```

Comment:
I'm realizing that file doesn't even seem necessary. I initially imagined it doing more things, but I'm currently in favor of deleting it and using websocket-stream directly.